packages/ui/Button.ts

```diff
@@ -0,0 +1,42 @@
+import { styled } from "styled-components";
+
+interface ButtonProps {
+  setPosition: string;
+  setIndex: string;
+};
+
+export const UIButton = styled.button<ButtonProps>`
```

Comment:
@MFelipeSilva please choose a better name to the button

Reply:
changes made.